plugins/modules/ndo_schema_template_bd_dhcp_policy.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Akini Ross (@akinross) <akinross@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_schema_template_bd_dhcp_policy
+short_description: Manage BD DHCP Policies in schema templates
+description:
+- Manage BD DHCP policies in schema templates on Cisco ACI Multi-Site.
+author:
+- Akini Ross (@akinross)
+options:
+  schema:
+    description:
+    - The name of the schema.
+    type: str
+    required: true
+  template:
+    description:
+    - The name of the template to change.
+    type: str
+    required: true
+  bd:
+    description:
+    - The name of the BD to manage.
+    type: str
+    required: true
+  dhcp_relay_policy:
+    description:
+    - The name of the DHCP Relay Policy.
+    type: str
+  dhcp_option_policy:
+    description:
+    - The name of the DHCP Option Policy.
+    - When the O(dhcp_option_policy) is provided, the O(dhcp_relay_policy) must also be provided.
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+notes:
+- This module can only be used on versions of NDO that are 4.x or greater.
```

Comment:
```suggestion
- This module can only be used on versions of NDO that are 4.1 or greater.
```
We don't support 4.0 so at least 4.1 here.

Reply:
We should also add a message on the older module that it shouldn't be used up to a certain version and to use this one for NDO 4.1+